instruction/db-sql/db-sql.md

```diff
@@ -88,20 +91,28 @@ Notice that each field is followed by the `NOT NULL` clause. That means each of
 
 The `id` field is also annotated with the `AUTO_INCREMENT` keyword. This means that you don't actually provide the `id` field when you insert a row. The database will do that for you using an automatically increasing integer.
 
+### Changing Tables
+
+> [!IMPORTANT]
+> Simply changing a "Create Table" statement stored in the code **does not** change the database. Read on to understand valid ways to change a SQL table.
```

Comment:
This also seems out of place for two reasons. 1) It is part of creating tables, not changing tables, 2) we haven't talked about creating tables in code yet.